composer.json

```diff
@@ -16,19 +16,20 @@
         }
     ],
     "minimum-stability": "dev",
+    "prefer-stable": true,
     "require": {
         "php": ">=5.3.9",
-        "symfony/framework-bundle": "~2.3.27 || ~2.6.6 || ~2.7 || ~3.0",
+        "symfony/framework-bundle": "^2.3.27 || ~3.0",
```

Comment:
this is semantically not the same as it allows 2.4, 2.5 and 2.6 < 2.6.6

i think there was a bugfix we needed that is in 2.3 but not 2.4 and 2.5 and only since 2.6.6. the correct thing is `~2.3.27 || ^2.6.6 || ~3.0`